docs/source/performance/caching.mdx

```diff
@@ -0,0 +1,225 @@
+  crew: [Astronaut]
+  designation: String!
+  startDate: String
+  endDate: String
+}
+
+extend type Astronaut @key(fields: "id") {
+  id: ID! @external
+  missions: [Mission]
+}
+
+type Query {
+  mission(id: ID!): Mission @cacheControl(maxAge: 10)
+  missions: [Mission]
+}
+```
+
+For the following query:
+
+```graphql
+query GetMissionWithCrew {
+  mission(id: 1) {
+    designation
+    crew {
+      name
+    }
+  }
+}
+```
+
+The response will be considered **not cacheable**.
+
+The `cache-control` header in the response from the astronauts subgraph will be `max-age=20, public` based on the cache hint applied to the `Astronaut` type, which will be taken into consideration when this subgraph responds to the `_entities` query from the gateway.
+
+The missions subgraph, however, will not include a `cache-control` header in its response to the gateway, thus indicating that it is not cacheable, and in turn, making the entire response sent to the client not cacheable. At first glance, this behavior may seem unexpected because the `@cacheControl(maxAge: 10)` directive is applied to the `mission` root query field. But upon closer inspection, we see that this field returns the `Mission` type which contains a non-scalar `Astronaut` field and this type will have a [default `maxAge` of `0` applied](https://www.apollographql.com/docs/apollo-server/performance/caching/#default-maxage).
+
+The missions subgraph isn't aware of the cache hint set on the `Astronaut` type in its owning subgraph, so there are two ways to address this. The first option is to apply an `@cacheControl` directive to the `Astronaut` entity extension with the `inheritMaxAge` argument set to `true`:
+
+```graphql{3}:title=Missions%20Subgraph
+type Mission {
+  id: ID!
+  crew: [Astronaut] @cacheControl(inheritMaxAge: true)
+  designation: String!
+  startDate: String
+  endDate: String
+}
+```
+
+Applying this argument will ensure that the `Astronaut` type inherits the `maxAge` set for its parent field (in this example, that's the `mission` field).
+
+The second option is to set an `@cacheControl` directive for the `Astronaut` entity extension with the appropriate `maxAge` value instead:
+
+```graphql{1}:title=Missions%20Subgraph
+extend type Astronaut @key(fields: "id") @cacheControl(maxAge: 20) {
+  id: ID! @external
+  missions: [Mission]
+}
+```
+
+Setting the `maxAge` property on the extended `Astronaut` type in the missions subgraph raises an important consideration about whether subgraphs should explicitly set these values for any entity types that they extend. In most cases, it will likely be preferable to apply `@cacheControl(inheritMaxAge: true)` wherever an extended entity type is used as a field's return type to avoid ambiguity. Depending on your requirements, you may wish to disallow this entirely using custom linting for your subgraphs to ensure that entity type extensions do not set the `maxAge` or `scope` arguments when applying the `@cacheControl` directive.
```

Comment:
"it will likely be preferable" -> "it's likely preferable"

"may wish" -> "might want"